<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Алгоритм Данцига</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 251, 251);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>191</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
            </property>
            <property name="text">
             <string>Предыдущая матрица смежности D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="before_num">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(170, 0, 0);</string>
            </property>
            <property name="text">
             <string>-1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="before">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>26</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>26</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>16</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Пользователь</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Введите размер матрицы (n x n):</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="size">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(249, 255, 255);</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="create">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string>Создать</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Нажмите, чтобы сгенерировать 
 случайную  матрицу смежности</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="generate">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Сгенерировать</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="next">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Следующая</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/image.jpg);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>151</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
            </property>
            <property name="text">
             <string>Новая матрица смежности D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="after_num">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(170, 0, 0);</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="after">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>26</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>26</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>16</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(170, 85, 255);</string>
          </property>
          <property name="text">
           <string>Короткие пути</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="short_track">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>16</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 170, 0);</string>
          </property>
          <property name="text">
           <string>Исходная матрица смежности D0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="initial">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1324</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
